concent_api/core/payments/bankster.py

```diff
@@ -292,38 +296,41 @@ def settle_overdue_acceptances(
             sum_of_existing_claims=Coalesce(Sum('amount'), 0)
         )
 
-        assert sum_of_existing_requestor_claims['sum_of_existing_claims'] >= 0
-
-        # If the existing claims against requestor's deposit are greater or equal to his current deposit,
-        # we can't add a new claim.
-        if requestor_deposit_value <= sum_of_existing_requestor_claims['sum_of_existing_claims']:
-            return None
-
         # Concent defines time T0 equal to oldest payment_ts from passed SubtaskResultAccepted messages from
         # subtask_results_accepted_list.
         oldest_payments_ts = min(subtask_results_accepted.payment_ts for subtask_results_accepted in acceptances)
 
-        # Concent gets list of transactions from payment API where timestamp >= T0.
-        list_of_transactions = service.get_list_of_payments(  # pylint: disable=no-value-for-parameter
+        already_satisfied_claims = get_already_satisfied_claims(
+            requestor_deposit_account,
+            provider_ethereum_address,
+            oldest_payments_ts,
```

Comment:
The only thing you do with `already_satisfied_claims` is pass it into `remove_duplicated_transactions_from_deposit_claims()`. You don't really need it here. Why not move it inside that function and rename the function to something like `find_unconfirmed_settlement_payments()`?

Reply:
:+1: done

Reply:
Is it done? I still see the old version here.

Reply:
Yes, it is done, you see old version because commented line doesn't get updated here. It is problematic sometimes because you have to go to file view to see if it was really fixed, but it is good that the code is kept here for reference of what was originally commented.

Reply:
No, I mean in the file view. I know that the snippets do not get updated. It was still visible in the code when I wrote that comment and this thread was not marked as "outdated'. I see that it has disappeared now. Maybe you did not push all the commits?